src/adaptors/index-coop/index.js

```diff
@@ -0,0 +1,36 @@
+const { default: BigNumber } = require('bignumber.js');
+const utils = require('../utils');
+
+const buildPool = (index, apy) => {
+  const chain = utils.formatChain(index.chain);
+  const symbol = utils.formatSymbol(index.symbol);
+  return {
+    pool: `${index.address}-${chain}`.toLowerCase(),
+    chain,
+    project: 'index-coop',
+    symbol,
+    tvlUsd: 0,
```

Comment:
Wasn't sure whether the TVL is required?

Reply:
it is required